### ISSUE TITLE
server url scoped to single paths

### ISSUE THREAD

**Issue by hanspagel** [scalar/scalar#1521]
Sometimes, you want to override the server URL just for a single path. We don’t support this yet, so let’s add support for it. :)

**Comment by marclave** [comment]
oh neat! is this part of the OpenAPI Specification?

**Comment by hanspagel** [comment]
Yes, it is. :) Path items can have “an alternative server array to service all operations in this path.”

https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#path-item-object

**Comment by antlio** [comment]
i believe this is fixed by #5335
